priv/emqx.schema

```diff
@@ -1007,12 +1007,6 @@ end}.
   {datatype, {duration, s}}
 ]}.
 
-%% @doc the number of smaples for calculate the average latency of delivery
-{mapping, "zone.$name.latency_samples", "emqx.zones", [
-  {default, 10},
-  {datatype, integer}
-]}.
```

Comment:
if we remove this from schema, old version config key will be come unknown
Please check how unknown config keys are handled, if error, we should keep it.